examples/use-cases/workflows/Quality Control Workflow for Quotes.ipynb

```
@@ -0,0 +1,2003 @@
+{
```

<blockquote>To resolve or ignore the exception, the user must supply an explanation in the <code>Details</code> field; this is ensured with the use of Guards. In either case, the relevant trigger is passed to the parent.</blockquote><p> </p><p>This is a great explanation overall, but may be worth expanding a little on what these 'guards' are, and some examples (e.g. can be used to set required fields). Would suggest adding another line expanding on the topic. I believe they use the LUSID filtering syntax, which would also not be obvious to a novice (or me!).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/753/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

---

<p>Extra info added</p>